etc/workflows/studio-upload.xml

```diff
@@ -0,0 +1,448 @@
+        id="include"
+        description="Include studio tracks normalization tasks">
+      <configurations>
+        <configuration key="workflow-id">partial-transcode-studio-tracks</configuration>
+      </configurations>
+    </operation>
+
+    <operation
+        id="snapshot"
+        fail-on-error="true"
+        exception-handler-workflow="partial-error"
+        description="Archiving">
+      <configurations>
+        <configuration key="source-tags">archive</configuration>
+      </configurations>
+    </operation>
+
+    <operation
+        id="prepare-av"
+        exception-handler-workflow="partial-error"
+        description="Preparing presenter (camera) audio and video work versions">
+      <configurations>
+        <configuration key="source-flavor">presenter/source</configuration>
+        <configuration key="target-flavor">presenter/prepared</configuration>
+        <configuration key="target-tags">-archive</configuration>
+        <configuration key="rewrite">false</configuration>
+        <configuration key="audio-muxing-source-flavors">*/?,*/*</configuration>
+      </configurations>
+    </operation>
+
+    <operation
+        id="prepare-av"
+        exception-handler-workflow="partial-error"
+        description="Preparing presentation (screen) audio and video work version">
+      <configurations>
+        <configuration key="source-flavor">presentation/source</configuration>
+        <configuration key="target-flavor">presentation/prepared</configuration>
+        <configuration key="target-tags">-archive</configuration>
+        <configuration key="rewrite">false</configuration>
+        <configuration key="audio-muxing-source-flavors">*/?,*/*</configuration>
+      </configurations>
+    </operation>
+
+    <!-- Create preview artifacts -->
+
+    <operation
+        id="include"
+        if="NOT ${skipEditorPreview}"
+        description="Prepare preview versions of the recording">
+      <configurations>
+        <configuration key="workflow-id">partial-preview</configuration>
+      </configurations>
+    </operation>
+
+    <operation
+        id="encode"
+        exception-handler-workflow="partial-error"
+        description="Encoding videos to delivery formats">
+      <configurations>
+        <configuration key="source-flavor">*/prepared</configuration>
+        <configuration key="target-flavor">*/delivery</configuration>
+        <configuration key="target-tags">engage-download,engage-streaming</configuration>
+        <configuration key="encoding-profile">studio.adaptive-parallel.http</configuration>
+      </configurations>
+    </operation>
+
+    <!-- Encode to engage player preview images -->
+
+    <operation
+        id="image"
+        exception-handler-workflow="partial-error"
+        description="Creating player preview image">
+      <configurations>
+        <configuration key="source-flavor">*/prepared</configuration>
+        <configuration key="source-tags"></configuration>
```

Comment:
```suggestion
        <configuration key="source-tags"></configuration>
```

Reply:
This is an copy paste artifact from existing workflow I've copied from, see https://github.com/opencast/opencast/blob/5144e24f9604df2f9aa3609c0936958c91e96f09/etc/workflows/partial-publish.xml#L157
We can remove it.